examples/minimal/tsconfig.json

```diff
@@ -1,19 +1,24 @@
 {
+  "$schema": "https://json.schemastore.org/tsconfig",
+  "_version": "20.1.0",
+
   "compilerOptions": {
```

Comment:
WDYT about extending main eslint/prettier/ts configs in example apps?

Reply:
Good idea!